.github/CODEOWNERS

```diff
@@ -9,14 +9,12 @@
 
 # File type specific owners
 # Markdown specific files
-*.md    @awentzel @chrisdholt @falkicon
+*.md @awentzel @chrisdholt @falkicon
 /docs/ @awentzel @chrisdholt @falkicon
 
 # Github management
 .github/ @awentzel @janechu @chrisdholt @nicholasrice
-
-# Circle Ci
-.circleci/ @awentzel @janechu @nicholasrice @chrisdholt
+.github/workflows/ @awentzel
```

Comment:
Can you add one more person to this path? We typically aim to have a minimum of 2 users assigned to any path.